extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -376,9 +378,27 @@ export class VisibilityManager {
           'elementHeight': layoutBox.height,
         }));
       }
+
+      // If spec includes initial scroll depth, set it
+      if (spec['initialScrollDepth']) {
+        model.maybeSetInitialScrollDepth(viewport.getScrollTop());
+        state['initialScrollDepth'] = model.getInitialScrollDepth();
+      }
+
+      if (spec['maxScrollDepth']) {
+        state['maxScrollDepth'] = model.getMaxScrollDepth();
+      }
+
       callback(state);
     });
 
+    // If spec includes max scroll depth, add scroll listener
+    if (spec['maxScrollDepth']) {
+      viewport.onScroll(() => {
+        model.maybeUpdateMaxScrollDepth(viewport.getScrollTop());
```

Comment:
Depending on what we want to return for `maxScrollDepth`, the value may be the same for all visibility models. If that's the case maybe we could have visibility manager keep one copy of the `maxScrollDepth` value.

Reply:
Done